tt_metal/impl/device/device_pool.cpp

```diff
@@ -163,7 +163,9 @@ void DevicePool::initialize_device(Device* dev) const {
     watcher_init(dev);
 
     // TODO: as optimization, investigate removing all this call for already initialized devivces
-    dev->reset_cores();
+    if (!llrt::OptionsG.get_skip_reset_cores_on_init()) {
```

Comment:
ugh, what a pita.  I don't like this - what's the flow?  if the system is unusable after init we have to know to set this env var and run again.  hmm.

Reply:
When we time out on waiting on clean reset, just continue on, and when we time out on fw init, prompt user to tt-smi reset

Reply:
Updated in latest commit